src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -794,86 +699,6 @@
     "src": "/compensation/special_benefit_allowances_2018.asp",
     "dest": "/disability/compensation-rates/special-benefit-allowance-rates/"
   },
-  {
```

Comment:
**First chunk**
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2015.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2014.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2013.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2012.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2011.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2009.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2008.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2007.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2006.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2005.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2004.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2003.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2002.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2001.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2000.asp 
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_1999.asp 

**Note: DON'T do anything with these (leave as is with redirects):**
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2019.asp
https://www.benefits.va.gov/compensation/special_Benefit_Allowances_2018.asp